cmd/pwp/main.go

```diff
@@ -127,14 +127,16 @@ func runServer(settings *siteSettings) error {
 	}
 
 	serve := exec.Command("php", "-S", fmt.Sprintf("%s:%s", *settings.host, *settings.port), "-t", *settings.path, *settings.path+"/router.php")
+	browse := exec.Command("open", fmt.Sprintf("http://%s:%s", *settings.host, *settings.port) )
```

Comment:
Open is also very OS specific. I think its fair to expect `php` to be installed if you want to run this, but `open` is not very useful in an environment that doesn't support it.

Reply:
I'm just lazy lol. will remove and update :)